jobs.json

```diff
@@ -1,5 +1,63 @@
 {
     "joblist": [
-
+        {
```

Comment:
I'm checking this into source control/git, but I'll un-check it in with a separate PR. That way everyone has this base-line list of jobs, but won't track it in the future